docs/admin/deployment/onion_name.rst

```diff
@@ -0,0 +1,108 @@
+Getting An Onion Name for Your SecureDrop
+-----------------------------------------
+
+What Are Onion Names?
+^^^^^^^^^^^^^^^^^^^^^
+
+Onion names are short, memorable addresses that visitors can use to access an 
+onion service (e.g., a news organization's SecureDrop) using Tor Browser. 
+
+Imagine a SecureDrop instance for a new organization called
+*The New York World* with a .onion address like this:
+
+`sdolvtfhatvsysc6l34d65ymdwxcujausv7k5jk4cy5ttzhjoi6fzvyd.onion`
+
+An onion name for this SecureDrop instance could be:
+
+`nyworld.securedrop.tor.onion`
+
+The general format for a SecureDrop onion name is:
+
+`<organization>.securedrop.tor.onion``
+
+How They Work
+^^^^^^^^^^^^^
+
+Onion names are supported in the desktop version of Tor Browser (introduced 
+in version 9.5). The mapping between onion names and the full-length .onion 
+addresses is done using a custom, signed ruleset for SecureDrop instances
+maintained by Freedom of the Press Foundation. The ruleset is updated
+automatically by Tor Browser, and no information is sent to third party servers
+when contacting a SecureDrop using an onion name.
+
+Onion names are currently not supported by the mobile version of Tor Browser,
+or by any other browser. (SecureDrop strongly recommends the use of the 
+desktop version of Tor browser.)
+
+The Tor project has committed to continued support of the onion name feature
+in some form. The underlying implementation and the address format may change
+in future iterations of this feature. To the extent that any changes are 
+required, we will reach out to coordinate them with you.
+
+Getting An Onion Name
+^^^^^^^^^^^^^^^^^^^^^
+
+Freedom of the Press Foundation maintains onion names for SecureDrop instances
+which:
+
+* are using v3 onion services
+* are part of the SecureDrop Directory
+
+We will generally approve onion names that meaningfully correspond to your name
+or that of your organization. Please note that, to disambiguate organizations
+in different countries with the same name, we may request the addition of a 
+country code (e.g. `<organization>.<country code>.securedrop.tor.onion`).
```

Comment:
(Noting that if we switch to preformatted above, we'll want to change it here as well.)